app/controllers/provider/admin/backend_apis_controller.rb

```diff
@@ -1,19 +1,16 @@
 # frozen_string_literal: true
 
 class Provider::Admin::BackendApisController < Provider::Admin::BaseController
-  before_action :find_backend_api, except: %i[new create]
-  before_action :authorize
+  load_and_authorize_resource :backend_api, through: :current_user, through_association: :accessible_backend_apis
```

Comment:
```suggestion
  load_and_authorize_resource :backend_api, through: :current_user, through_association: :accessible
```

And no need to create an association for this

https://github.com/CanCanCommunity/cancancan/blob/2.3.0/lib/cancan/controller_resource_loader.rb#L67

Reply:
I'm afraid it doesn't work like this. What you suggested will invoke `accessible` on `current_user`, resulting in
```
NoMethodError: undefined method `accessible' for #<User:xxx>
```